reporting.md

```diff
@@ -327,35 +328,39 @@ The JSON schema is derived from the HPKP JSON schema [@?RFC7469] (cf. Section 3)
 
 ```
 {
-  "organization-name": organization-name,
+  "organization-name": "organization - name",
```

Comment:
Why this change (and those below)? I mean, why add quotes and all?

Reply:
To put the strings in quotes?  AFAIK, that's how JSON expects them to be, and I would think someone wouldn't want to put a bare string after the colon.  It can be reverted.

Reply:
Agreed. But this is a schema, not an example report. I thought it made sense that "date-time" and "email-address" and so forth were variables standing in for the real thing. That's how it's written in the referenced https://tools.ietf.org/html/rfc7469#section-3.

Reply:
Ugh, crap, I thought you were commenting on the sample.  Yeah, we can remove those quotes.